.github/ISSUE_TEMPLATE/api-bug-report.yml

```diff
@@ -0,0 +1,50 @@
+name: API Bug Report
+description: A bug has been found in Discord's API, Gateway, OAuth2, SDK, etc.
+title: 'bug'
```

Comment:
The one downside to the forms approach is they require a `title` attribute (non-empty) - this is pre-filled in to the issue's title. Certainly open to better suggestions if y'all have specific verbiage you'd like there.

Reply:
Does `title: ""` not work?

Reply:
You can leave the title fields. Like only name and description